bcgov/moh-prime
# PRIME-2113 can't delete enrollees

[PRIME-2113](https://bcgovmoh.atlassian.net/browse/PRIME-2113)

### How to test:

1. Create enrollee
2. Change status/add notes
3. Delete enrollee (should be deleted)
